extensions/standard-processors/processors/HashContent.h

```diff
@@ -49,21 +48,25 @@ namespace { // NOLINT
     HashReturnType ret_val;
     ret_val.second = 0;
     std::array<std::byte, HASH_BUFFER_SIZE> buffer{};
-    MD5_CTX context;
-    MD5_Init(&context);
+    EVP_MD_CTX *context = EVP_MD_CTX_new();
+    const auto guard = gsl::finally([&context]() {
+      EVP_MD_CTX_free(context);
+    });
+    const EVP_MD *md = EVP_md5();
+    EVP_DigestInit_ex(context, md, nullptr);
```

Comment:
Is there any reason not to inline the digest type call?
```suggestion
    EVP_DigestInit_ex(context, EVP_md5(), nullptr);
```

Reply:
I can be inlined, updated in 5ebb285e42f5ea89391d5b126bcbecb2b27c77d9